core/controllers/skill_editor_test.py

```diff
@@ -165,6 +169,88 @@ def test_editable_skill_handler_get_succeeds(self):
         # Check that admins can access the editable skill data.
         json_response = self.get_json(self.url)
         self.assertEqual(self.skill_id, json_response['skill']['id'])
+        self.assertEqual(
+            json_response['assigned_skill_topic_data_dict']['Name'],
+            'Subtopic1')
+        self.assertEqual(
+            1, len(json_response['grouped_skill_summaries']['Name']))
+        self.logout()
+
+    def test_skill_is_assigned_to_topic_but_not_subtopic(self):
```

Comment:
test_skill_which_is_assigned_to_...

(because otherwise, it sounds like you're testing whether the assignment of the skill is happening correctly. This is not the case here.)

Ditto below.

Reply:
Done